docs/resources/link_level_flow_control_interface_policy.md

```diff
@@ -0,0 +1,126 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Access Policies"
+layout: "aci"
+page_title: "ACI: aci_link_level_flow_control_interface_policy"
+sidebar_current: "docs-aci-resource-aci_link_level_flow_control_interface_policy"
+description: |-
+  Manages ACI Link Level Flow Control Interface Policy
+---
+
+# aci_link_level_flow_control_interface_policy #
+
+Manages ACI Link Level Flow Control Interface Policy
+
+
+
+## API Information ##
+
+* Class: [qosLlfcIfPol](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/qosLlfcIfPol/overview)
+
+* Supported in ACI versions: 4.2(6d) to 4.2(7w), 5.1(1h) and later.
+
+* Distinguished Name Format: `uni/infra/llfc-{name}`
+
+## GUI Information ##
+
+* Location: `Fabric -> Access Policies -> Policies -> Interface -> Link Level Flow Control`
+
+## Example Usage ##
+
+The configuration snippet below creates a Link Level Flow Control Interface Policy with only required attributes.
+
+```hcl
+
+resource "aci_link_level_flow_control_interface_policy" "example" {
+  name = "test_name"
+}
+
+```
+The configuration snippet below shows all possible attributes of the Link Level Flow Control Interface Policy.
+
+!> This example might not be valid configuration and is only used to show all possible attributes.
+
+```hcl
+
+resource "aci_link_level_flow_control_interface_policy" "full_example" {
+  annotation   = "annotation"
+  description  = "description_1"
+  receive_mode = "on"
+  send_mode    = "on"
+  name         = "test_name"
+  name_alias   = "name_alias_1"
+  owner_key    = "owner_key_1"
+  owner_tag    = "owner_tag_1"
+  annotations = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+  tags = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+}
+
+```
+
+All examples for the Link Level Flow Control Interface Policy resource can be found in the [examples](https://github.com/CiscoDevNet/terraform-provider-aci/tree/master/examples/resources/aci_link_level_flow_control_interface_policy) folder.
+
+## Schema ##
+
+### Required ###
+
+* `name` (name) - (string) The name of the Link Level Flow Control Interface Policy object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the Link Level Flow Control Interface Policy object.
+
+### Optional ###
+
+* `annotation` (annotation) - (string) The annotation of the Link Level Flow Control Interface Policy object.
+  - Default: `"orchestrator:terraform"`
+* `description` (descr) - (string) The description of the Link Level Flow Control Interface Policy object.
+* `receive_mode` (llfcRcvAdminSt) - (string) The receive mode of the Link Level Flow Control Interface Policy object.
```

Comment:
Should these be receive state or similar? To me using mode implies there are multiple options instead of on or off.
The UI says Receive/Send Flow Control. I think either receive_flow_control or receive_state?